core/src/main/scala/scala/pickling/Ops.scala

```diff
@@ -23,6 +23,7 @@ final case class UnpickleOps(val thePickle: Pickle) {
     functions.unpickle(thePickle.asInstanceOf[format.PickleType])(unpickler, format)
 }
 
+/** A layer of pickling cake that can "lift" picklable types to have the core pickle operations. */
```

Comment:
Term *protocol stack* might be easier to understand than *cake*. That's the term we used on README.
Also link to `[[scala.pickling.Defaults]]` as an example? https://github.com/scala/pickling/blob/0.11.x/core/src/main/scala/scala/pickling/Defaults.scala